src/SqlPersistence/Saga/SqlSagaAttribute.cs

```diff
@@ -0,0 +1,46 @@
+﻿using System;
+
+namespace NServiceBus.Persistence.Sql
+{
+    /// <summary>
+    /// Exposes extra configuration options for storing <see cref="Saga{TSagaData}"/>.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Class, Inherited = false)]
+    public sealed class SqlSagaAttribute : Attribute
```

Comment:
Maybe I misunderstood but if we're using core saga then we shouldn't have access to those "sqlsaga-specific extras", correct?

I thought this attribute will be the (potential) next step after we implement changes in core saga to converge mappings etc.

Reply:
I don't see any reason to not support transitional correlation id on a core saga when it's this easy to implement. SqlSaga is a way to make correlation id mapping more DRY. I don't see a reason to overload it.